ios-continuous-integration_5c8a397c2c7d3a154460db00.md

```diff
@@ -123,228 +122,218 @@ blocks:
 
 ``` yaml
-        commands:
-          - bundle exec fastlane build
+commands:
+  - bundle exec fastlane build
 ```
 
 ### Fastfile
 
-The example is using the Semaphore Fastlane plugin:
+The example project uses the [Semaphore fastlane plugin](fastlane-plugin):
 
-```ruby
-# fastlane/Fastfile
+``` ruby
 default_platform(:ios)
 
-platform :ios do
+before_all do
+  # Install with `fastlane add_plugin semaphore`
+  setup_semaphore
+end
 
-  before_all do
-    # installed via the semaphore plugin with `fastlane add_plugin semaphore`
-    setup_semaphore
+platform :ios do
+  lane :build do
+    gym(scheme: 'TallestTowers',
+        skip_package_ipa: true,
+        skip_archive: true,
+        clean: true)
   end
 
-  desc "Run Tests"
   lane :test do
-    scan
-  end
-
-  desc "Build"
-  desc "Build without code sign. Just to see if the build is working"
-  lane :build do |options|
-    gym(
-      scheme: "HelloWorld",
-      skip_package_ipa: true,
-      skip_archive: true,
-      silent: true,
-      clean: true
-    )
+    run_tests(scheme: 'TallestTowers',
+              devices: ['iPhone 8', 'iPhone 11 Pro'])
   end
 end
 ```
 
 ### Releasing your app
 
-To manage your developer credentials on Semaphore, use [encrypted
-secrets][secrets].
-
-To manage releases, [set up promotions][promotions] to trigger additional pipelines.
-
-Related guides:
-
-- [Code signing for iOS projects][code-signing]
-- [TestFlight integration][testflight]
-- [HockeyApp integration][hockeyapp]
+To manage your developer credentials on Semaphore, use
+[encrypted secrets][encrypted-secrets]. To manage releases,
+[set up promotions][promotions] to trigger additional pipelines.
 
 ## Run the demo project yourself
 
 A good way to start using Semaphore is to take a demo project and run it
 yourself. Here’s how to build the demo project with your own account:
 
-1. [Fork the project on GitHub][demo-project] to your own account.
+1. [Fork the project on GitHub][example-project] to your own account.
 2. Clone the repository on your local machine.
 3. In Semaphore, follow the link in the sidebar to create a new project.
 4. Create your secret as per instructions above.
 5. Edit any file and push GitHub, and Semaphore will run the CI/CD pipeline.
 
-[macos-mojave]: https://docs.semaphoreci.com/article/161-macos-mojave-xcode-10-image
+### Related guides:
+
+- [Code signing for iOS projects][code-signing]
+- [TestFlight integration][testflight]
+- [HockeyApp integration][hockeyapp]
+
+[macos-mojave-xcode10]: https://docs.semaphoreci.com/article/161-macos-mojave-xcode-10-image
+[macos-mojave-xcode11]: https://docs.semaphoreci.com/article/162-macos-mojave-xcode-11-image
 [machine-types]: https://docs.semaphoreci.com/article/20-machine-types
-[beta-apply]: https://semaphoreci.com/product/ios
-[demo-project]: https://github.com/semaphoreci-demos/semaphore-demo-ios-swift-xcode
+[example-project]: https://github.com/semaphoreci-demos/semaphore-demo-ios-swift-xcode
```

Comment:
Should be the new repo, https://github.com/semaphoreci-demos/semaphore-demo-ios-swift-xcode-v2